frontend/test/metabase/scenarios/filters/filter-bulk.cy.spec.js

```diff
@@ -514,23 +504,23 @@ describe("scenarios > filters > bulk filtering", () => {
 
   describe("text filters", () => {
     beforeEach(() => {
-      visitQuestionAdhoc(productsQuestion);
+      visitQuestionAdhoc(peopleQuestion);
       openFilterModal();
     });
 
     it("adds a contains text filter", () => {
       modal().within(() => {
-        cy.findByLabelText("Title").within(() => {
-          cy.findByPlaceholderText("Search by Title").type("Marble");
+        cy.findByLabelText("City").within(() => {
```

Comment:
same problem with https://github.com/metabase/metabase/pull/23403/files#r908199201, so change to use People.City, which were not cached.